static/js/filter_country.js

```diff
@@ -184,4 +184,18 @@ function clearFilters() {
 		filter.style.visibility = "visible"
 	});
 
+	clearSelected("feature");
 }
+
+function clearSelected(domId){
+	domElement = document.getElementById(domId)
+	domElement.value = ""
```

Comment:
What's the use of this attribute?

Reply:
domElement.value is the value that will be send by the url when submitting the form